src/assets/scss/global.scss

```diff
@@ -0,0 +1,2 @@
+@import url('https://fonts.googleapis.com/css2?family=Roboto:wght@100;300;400;500;700;900&display=swap');

```

Comment:
we are already using web safe fonts so we dont need google fonts.

Reply:
This line has been removed.